src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsKruxObject.java

```diff
@@ -49,17 +46,23 @@ public void verifyKruxControlTag(String kruxSiteId) {
    * Test whether the Krux user id is not empty and added to GPT calls
    */
   public void verifyKruxUserParam() {
+    String script = "return localStorage.kxuser;";
+    waitForKrux();
+    String user1 = (String) ((JavascriptExecutor) driver).executeScript(script);
```

Comment:
How about adding a `TODO:` comment? We should discuss it and if possible find out why it happens...